packages/razzle-plugin-scss/helpers.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+const makeLoaderFinder = require('razzle-dev-utils/makeLoaderFinder');
+
+const cssLoaderFinder = makeLoaderFinder('css-loader');
+const postCssLoaderFinder = makeLoaderFinder('postcss-loader');
+const resolveUrlLoaderFinder = makeLoaderFinder('resolve-url-loader');
+const sassLoaderFinder = makeLoaderFinder('sass-loader');
+const styleLoaderFinder = makeLoaderFinder('style-loader');
+
+module.exports = {
+  cssLoaderFinder,
+  postCssLoaderFinder,
+  resolveUrlLoaderFinder,
+  sassLoaderFinder,
+  styleLoaderFinder,
```

Comment:
Should we document in README what loaderFinders are being exported so people can reuse them?

Reply:
It would be a good idea! I don't know whether some loaders are reusable at first